fastai/data/external.py

```diff
@@ -110,15 +112,24 @@ class URLs():
     WT103_FWD          = f'{S3_MODEL}wt103-fwd.tgz'
     WT103_BWD          = f'{S3_MODEL}wt103-bwd.tgz'
 
-    def path(url='.', c_key='archive'):
+    def path(
```

Comment:
Add types to all param lists and return types where possible (as long as they are simple types)

Reply:
Done